content/en/real_user_monitoring/android/mobile_vitals.md

```diff
@@ -15,13 +15,17 @@ further_reading:
 
 Real User Monitoring offers Mobile Vitals, a set of metrics inspired by [Android Vitals][1], that can help compute insights about your mobile application's responsiveness, stability, and resource consumption. Mobile Vitals range from poor, moderate, to good.
```

Comment:
```suggestion
Mobile Vitals range from poor, moderate, to good.
```

Is this part still true?

Reply:
Yes I believe so! CC @timmld just in case